masonry_core/src/app/render_root.rs

```diff
@@ -216,23 +216,33 @@ pub enum RenderRootSignal {
     RequestAnimFrame,
     /// The window should take focus.
     TakeFocus,
-    /// The mouse icon has changed.
+    /// The mouse icon should change.
     SetCursor(CursorIcon),
-    /// The window size has changed.
+    /// The window should be resized.
     SetSize(PhysicalSize<u32>),
     /// The window title has changed.
     SetTitle(String),
-    /// The window is being dragged.
+    /// The user has started dragging the window.
+    ///
+    /// Masonry should send this event when the user presses the left mouse button while hovering a client-side decoration representing the window title bar or similar.
+    /// The platform that receives this event should start moving the window around until the mouse button is released.
     DragWindow,
-    /// The window is being resized.
+    /// The user has started drag-resizing the window.
+    ///
+    /// Masonry should send this event when the user presses the left mouse button while hovering a client-side decoration representing a window resize handle.
+    /// The platform that receives this event should start resizing the window until the mouse button is released.
     DragResizeWindow(ResizeDirection),
-    /// The window is being maximized.
+    /// The window should be maximized.
     ToggleMaximized,
-    /// The window is being minimized.
+    /// The window should be minimized.
     Minimize,
-    /// The window is being closed.
+    /// The app should terminate.
     Exit,
-    /// The window menu is being shown.
+    /// The [Windows system menu] should be shown.
+    ///
+    /// Other platforms may ignore this.
+    ///
+    /// [Windows system menu]: https://en.wikipedia.org/w/index.php?title=Common_menus_in_Microsoft_Windows&oldid=1285312933#System_menu
```

Comment:
This is confusing to me. I thought this was mostly added to Masonry for the Wayland thing - https://wayland.app/protocols/xdg-shell#xdg_toplevel:request:show_window_menu.

It's documented as unsupported on Wayland in Winit, but it is, at least on main - https://github.com/rust-windowing/winit/blob/120f21a010b6831bcbb75925bc11d4dcb07dd645/winit-wayland/src/window/state.rs#L907.